tools/IoTEdgeDevOps/DevOpsLib/ValidationUtil.cs

```diff
@@ -22,6 +22,14 @@ public static void ThrowIfNonPositive(int value, string field)
                 throw new ArgumentOutOfRangeException(field, "Cannot be less than or equal to zero.");
             }
         }
+        public static void ThrowIfNegative(int value, string field)
```

Comment:
Should this function also check for Null?

Reply:
int type cannot be null.  but i will use string.empty if field is null.